source/agora/consensus/protocol/Nominator.d

```diff
@@ -324,10 +324,6 @@ extern(D):
 
```

Comment:
I don't think this is correct. The node itself starts a periodic timer which recursively calls itself until nomination is over. See here: https://github.com/bpfkorea/agora/blob/b6c7b12b9c8dc3fbc88f6e3f8c8dbefd29af5a08/source/scpp/src/scp/NominationProtocol.cpp#L522-L528

Also if this was a bugfix it should have come with a test-case where the test fails without the bugfix but succeeds with the bugfix.

Reply:
Btw, in the next commit this change seems to be reverted.